greet.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
+
+var Greet = module.exports = exports = function () {};
+Greet.prototype.greet = function (name) {
+  return "Hello " + name;
+};
```

Comment:
No need to prototype the greet function. Also, you should just be exporting an object, not an object constructor function. Module.exports = { greet: function(name){ return "hello "  + name }}